Extension/CHANGELOG.md

```diff
@@ -14,18 +14,21 @@
 * Change the `Disabled` value for `C_Cpp.errorSquiggles` to stop showing missing header squiggles. [#3361](https://github.com/Microsoft/vscode-cpptools/issues/3361)
 * Add a new (default) value of `EnabledIfIncludesResolve` to `C_Cpp.errorSquiggles`, which only shows error squiggles if include headers are successfully resolved. [PR #3421](https://github.com/Microsoft/vscode-cpptools/pull/3421)
 * Enable flag `/permissive-` as an argument to `compilerPath` with `cl.exe`. [#3446](https://github.com/Microsoft/vscode-cpptools/issues/3446)
+* Disable debug heap by default with cppvsdbg. [#3484](https://github.com/Microsoft/vscode-cpptools/issues/3484)
```

Comment:
Add subpoint "Reported by Djoulihen"

Reply:
Hmm, we don't usually mention "reported by", but if you think this warrants that, it seems fine to me.

Reply:
That's true, its usually for people who contribute PR's. 

I'm ok withholding it, and we already have the issue linked.